app/payment-request/index.js

```diff
@@ -1,6 +1,6 @@
 const getExistingPaymentRequest = require('./get-existing-payment-request')
 const getPaymentRequestCount = require('./get-payment-request-count')
-const { getPaymentRequest, getPaymentRequestByInvoiceNumberAndRequestId, getPaymentRequestByRequestId } = require('./get-payment-request')
+const { getPaymentRequest, getPaymentRequestByInvoiceNumberAndRequestId, getPaymentRequestByRequestId, getPaymentRequestAwaitingEnrichment } = require('./get-payment-request')
```

Comment:
Could the order of this be changed to 'getPaymentRequest, getPaymentRequestAwaitingEnrichment, getPaymentRequestByInvoiceNumberAndRequestId, getPaymentRequestByRequestId' to loosely match the order below?